.circleci/config.yml

```
@@ -25,10 +18,10 @@ commands:
 jobs:
   lint:
     docker:
-      - image: meliuz/docker-linter:1.2.0
-        auth:
-          username: $DOCKER_USER
-          password: $DOCKER_PASS
+      - image: $AWS_ECR_URI/docker-linter:v1.12.0
```

Aqui não poderia usar a versão `latest`?

```suggestion
      - image: $AWS_ECR_URI/docker-linter:latest
```